src/web.cpp

```diff
@@ -1057,7 +1060,7 @@ void handle_update()
         server.send_P(200, type_txt, PSTR("Update Success! Rebooting...\n"));
         delay(100);
         server.stop();
-        sync_and_restart();
+        //sync_and_restart();
```

Comment:
for now we are not rebooting after an upload.  this allows a second call to verify and then a manual call to the reboot api.

Reply:
Item 1 is wrong for a lot of reasons.  2 copies to flash instead of 1, you still have to verify the first copy. The problem with 2, is if a write somehow corrupts a previous block that has already been verified.

Once we root cause the OTA failures (and other flash corruptions), we can change the verify to just read the contents of the flash and recheck the MD5.  There should be no reason to do this byte by byte comparison.  However, until we root cause, I want to keep it like this.  We can revert this before we merge.

Reply:
okay, then we can clean things up... let me work on the javascript side of things so it will report progress to user for each step.  Would you like an automatic reboot after the verification, or positive action from the user (press a reboot-now button).